fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PauseRoutineLoadAnalyzer.java

```diff
@@ -0,0 +1,32 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.analyzer;
+
+import com.google.common.base.Strings;
+import com.starrocks.analysis.LabelName;
+import com.starrocks.analysis.PauseRoutineLoadStmt;
+import com.starrocks.cluster.ClusterNamespace;
+import com.starrocks.common.ErrorCode;
+import com.starrocks.common.ErrorReport;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.server.CatalogMgr;
+
+public class PauseRoutineLoadAnalyzer {
+    public static void analyze(PauseRoutineLoadStmt statement, ConnectContext context) {
+        String db = statement.getDbFullName();
+        String catalog = context.getCurrentCatalog();
+        if (Strings.isNullOrEmpty(db)) {
+            db = context.getDatabase();
+            if (CatalogMgr.isInternalCatalog(catalog)) {
+                db = ClusterNamespace.getFullName(db);
+            }
+            if (Strings.isNullOrEmpty(db)) {
+                ErrorReport.reportSemanticException(ErrorCode.ERR_NO_DB_ERROR);
```

Comment:
Please write a clear error message like `database xxx not exists`

Reply:
this message is com.starrocks.sql.analyzer.SemanticException: No database selected
user did not select a database instead of selected database does not exist
Please help to confirm， thanks

for example 
1. PAUSE ROUTINE LOAD FOR label ; fail and throw message (No database selected)
2. use testdb;  PAUSE ROUTINE LOAD FOR label ;  success
3. PAUSE ROUTINE LOAD FOR testdb.label ;  success

Reply:
got it.